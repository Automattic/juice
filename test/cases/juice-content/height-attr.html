<html>
<head>
<style>
    p, td.px, th.px, img.px {
        height: 200px;
    }
    td.percentage, th.percentage, img.percentage {
        height: 50%;
    }
    td.auto, th.auto, img.auto {
        height: auto;
    }
</style>
</head>
<body>
    <p>none</p>
    <table>
        <tr>
            <th class="px">
                header 1
            </th>
            <th class="percentage">
                header 2
            </th>
            <th class="auto">
                header 3
            </th>
        </tr>
        <tr>
            <td class="px">
                high
            </td>
            <td class="percentage">
                high
            </td>
            <td class="auto">
                high
            </td>
        </tr>
    </table>
    <img src="#" alt="high" class="px">
    <img src="#" alt="high" class="percentage">
    <img src="#" alt="high" class="auto">
</body>
</html>
