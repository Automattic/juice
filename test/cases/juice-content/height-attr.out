<html>
<head>

</head>
<body>
    <p style="height: 200px;">none</p>
    <table>
        <tr>
            <th class="px" style="height: 200px;" height="200">
                header 1
            </th>
            <th class="percentage" style="height: 50%;" height="50%">
                header 2
            </th>
            <th class="auto" style="height: auto;" height="auto">
                header 3
            </th>
        </tr>
        <tr>
            <td class="px" style="height: 200px;" height="200">
                high
            </td>
            <td class="percentage" style="height: 50%;" height="50%">
                high
            </td>
            <td class="auto" style="height: auto;" height="auto">
                high
            </td>
        </tr>
    </table>
    <img src="#" alt="high" class="px" style="height: 200px;" height="200">
    <img src="#" alt="high" class="percentage" style="height: 50%;">
    <img src="#" alt="high" class="auto" style="height: auto;" height="auto">
</body>
</html>
